File: java-spring/src/main/java/com/example/javaspring/DefaultData.java
package com.example.javaspring;

import com.example.javaspring.service.UserService;
import com.example.javaspring.service.command.CreateUser;
import org.springframework.stereotype.Service;

import java.util.ArrayList;
import java.util.Arrays;

@Service
public class DefaultData {

    private final UserService userService;

    public DefaultData(UserService userService) {
        this.userService = userService;
    }

    ArrayList<String> nameList = new ArrayList<>(Arrays.asList(
            "jeong", "jinu", "jinwoo", "zinu", "henry", "liha", "mingsu", "cloud",
            "bunny bernie", "no more build", "woori", "stevy", "vegi"
    ));

    public void init() {
        for (String personName : nameList) {
            userService.createUser(new CreateUser(personName, (Long) Math.round(Math.random() * 120)));
        }
    }
}

File: kotlin-spring/src/main/kotlin/com/example/kotlinspring/DefaultData.kt
package com.example.kotlinspring

import com.example.kotlinspring.service.UserService
import com.example.kotlinspring.service.command.CreateUser
import org.springframework.stereotype.Service
import kotlin.math.roundToLong

@Service
class DefaultData(
    private val userService: UserService
){
    val nameList = listOf(
        "jeong", "jinu", "jinwoo", "zinu", "henry", "liha", "mingsu", "cloud",
        "bunny bernie", "no more build", "woori", "stevy", "vegi",
        )

    fun init() {
        for (personName in nameList){
            userService.createUser(CreateUser(personName, (Math.random() * 120).roundToLong()))
        }
    }
}